### PR TITLE
Implement default layout

### PR DESCRIPTION
**Pull request by moetazaneta**

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ */
+export const Header: VFC = memo(() => (
+  <header className={cls.header}>
+    <div className={cls.search}>
+      <Icon name="carbon:search" />
+      <span>Search</span>
+    </div>
+    <div className={cls.profile}>
+      <Icon name="carbon:user-avatar" size={32} />
+      <span> Profile</span>
```
Comment (Moltenship) [inline src/components/Header/Header.tsx]: Remove space ```<span>Profile</span>```

File: src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  {
+    type: SidebarElementType.Heading,
+    text: 'vndb re',
+    items: [
+      { type: SidebarElementType.Link, text: 'Source code', link: '#' },
+      { type: SidebarElementType.Link, text: 'Discord', link: '#' },
+    ],
+  },
+  { type: SidebarElementType.Heading, text: 'Statistics', items: [] },
+];
```
Comment (Moltenship) [inline src/components/Sidebar/Sidebar.tsx]: Move it to another file

File: src/components/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          return (
+            <>
+              <Heading className={cls.heading}>{el.text}</Heading>
+              {el.items.map(_el => <a href={_el.link} className={cls.link}>{_el.text}</a>)}
+            </>
+          );
+        }
+        case SidebarElementType.Link: {
+          return <a href={el.link} className={cls.link}>{el.text}</a>;
+        }
```
Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.tsx]: Use `<Link>` component for navigation between internal pages of our app

File: src/components/Sidebar/Sidebar.module.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  font-size: 32px;
+  font-weight: 900;
+  flex-shrink: 0;
+  height: 64px;
+  display: flex;
+  align-items: center;
+}
+
+.heading {
+  margin-top: 12px;
```
Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.module.css]: Can you use charkra variable for spacing here?

File: src/components/Sidebar/Sidebar.module.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  font-size: var(--chakra-fontSizes-md) !important;
```
Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.module.css]: I think it's better to set up font size through `size` prop of `Heading` component

File: src/components/Sidebar/Sidebar.tsx (hunk 3 above)
Comment (NightOfNights) [reply src/components/Sidebar/Sidebar.tsx:120]: For internal navigation

Comment (NightOfNights) [reply src/components/Sidebar/Sidebar.tsx]: Also you can use `<Navlink>` instead of `<Link>` to apply styles to active link

Comment (Moltenship) [reply src/components/Sidebar/Sidebar.tsx]:
```js
<Link as={NavLink} to={el.link}>
```
`Link` component comes from chakra-ui and `NavLink` comes from react-router-dom.

Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.tsx]: You have error there with keys (I checked it on my local machine). Pls fix it

**Review by NightOfNights — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/Sidebar.tsx [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+        default: {
+          return null;
+        }
+      }
+    })}
```
Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.tsx]:
```suggestion
    {data.map(el => {
      switch (el.type) {
        case SidebarElementType.Heading: {
          return (
            <Fragment key={el.text + el.type}>
              <Heading className={cls.heading}>{el.text}</Heading>
              {el.items.map(_el => <a href={_el.link} key={_el.link + _el.text} className={cls.link}>{_el.text}</a>)}
            </Fragment>
          );
        }
        case SidebarElementType.Link: {
          return <a href={el.link} key={el.link + el.text} className={cls.link}>{el.text}</a>;
        }
        default: {
          return null;
        }
      }
    })}
```

Comment (NightOfNights) [reply src/components/Sidebar/Sidebar.tsx]: add import { Fragment } from 'react';

**Review by prekel — COMMENTED** [review]
image

While loading, there is no space between panel and search. Or it will be fixed in #54?

**Review by prekel — CHANGES_REQUESTED** [review]
test

**Review by prekel — APPROVED** [review]

File: src/components/Header/Header.tsx [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React, { memo, VFC } from 'react';
+import { Icon } from '../Icon/Icon';
+import cls from './Header.module.css';
+
+/**
+ * TODO: Implement header funtionality.
+ */
+export const Header: VFC = memo(() => (
+  <header className={cls.header}>
```
Comment (NightOfNights) [inline src/components/Header/Header.tsx]: Don't wrap components without props in `memo`.

File: src/components/Sidebar/Sidebar.tsx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+import React, { Fragment, memo, VFC } from 'react';
+import { NavLink } from 'react-router-dom';
+import cls from './Sidebar.module.css';
+import { data, SidebarElementType } from './data';
+
+/**
+ * Sidebar.
+ */
+export const Sidebar: VFC = memo(() => (
+  <aside className={cls.sidebar}>
```
Comment (NightOfNights) [inline src/components/Sidebar/Sidebar.tsx]: Same

File: src/components/Header/Header.module.css [hunk 9]
```diff
@@ -0,0 +1,28 @@
+.header {
+  border-bottom: 1px solid var(--chakra-colors-gray-300);
+  height: 64px;
```
Comment (Moltenship) [inline src/components/Header/Header.module.css:3]: Let's add new design tokens for 8/16/32/64px sizes. It seems we use this a lot.

**Review by Moltenship — APPROVED** [review]

File: src/components/Header/Header.module.css (hunk 9 above)
Comment (moetazaneta) [reply src/components/Header/Header.module.css:3]: no.

**Review by NightOfNights — APPROVED** [review]

**Pull request merged**